source/Glimpse.Site/Glimpse.Site.csproj

```diff
@@ -20,6 +20,7 @@
     <IISExpressAnonymousAuthentication />
     <IISExpressWindowsAuthentication />
     <IISExpressUseClassicPipelineMode />
+    <ApplicationInsightsResourceId>/subscriptions/72993b69-db12-44fc-9a66-9c2005c30513/resourcegroups/Default-ApplicationInsights-CentralUS/providers/microsoft.insights/components/Glimpse.Site</ApplicationInsightsResourceId>
```

Comment:
please remove this line as it will point to the resource in your subscription. It will be out of sync with InstrumentationKey almost right away.